dmoj/cptbox/_cptbox.pyx

```diff
@@ -341,13 +344,15 @@ cdef class Process:
     cdef public unsigned int _cpu_time
     cdef public int _nproc
     cdef unsigned long _max_memory
+    cdef bytes _re_fs_read
```

Comment:
I don't really like this naming scheme, took several attempts before I parsed it correctly. `_re_readable_fs`?